### PR TITLE
6.1 - Update travis to model

### PR DESCRIPTION
**Pull request by bwrsandman**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,24 @@
 language: python
+
 python:
   - "2.7"
 
 env:
   - VERSION="6.1" ODOO_REPO="odoo/odoo"
   - VERSION="6.1" ODOO_REPO="OCA/OCB"
```
Comment (pedrobaeza) [inline .travis.yml]: There is no 6.1 OCB version on Github, so you must remove this one

**Comment by bwrsandman** [comment]
Test fails are unrelated

**Comment by pedrobaeza** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**